.github/CODEOWNERS

```diff
@@ -13,23 +13,16 @@
 /client/my-sites/comment* @Automattic/serenity
 
 # Email services
-/client/components/data/query-gsuite-users @Automattic/yolo
-/client/components/domains/contact-details-form-fields/index.jsx @Automattic/yolo
 /client/components/gsuite @Automattic/yolo
 /client/components/marketing-survey/gsuite-cancel-purchase-dialog @Automattic/yolo
 /client/lib/domains/email-forwarding @Automattic/yolo
 /client/lib/gsuite @Automattic/yolo
 /client/lib/titan @Automattic/yolo
-/client/me/purchases/remove-purchase/index.jsx @Automattic/yolo
```

Comment:
Not related to email services. It only loads gsuite cancel dialog component, which already has its own code owner entry.